### ISSUE TITLE
When selecting multiple calendars the same event shouldn't show up multiple times

### ISSUE THREAD

**Issue by Defcon0** [nextcloud/calendar#1449]
### Steps to reproduce
1. Create event in calendar 1
2. Add participant to that event (member 2 with calendar 2)
3. Now select calendar 1 and calendar 2 in my list.

### Expected behaviour
The event in which member 1 and member 2 are participants of is shown only once because it's the same event.

### Actual behaviour
The event in which member 1 and member 2 are participants of is shown multiple times because it's in both selected calendars.

This shouldn't be. Imagine a team event and having selected all team calendars in order to be kept track of the team's events (even though I might not be participant of all of them). In this case an event would be shown 20 times for example.

### Server configuration
**Operating system**: Linux

**Web server:** Apache

**Database:** MySQL

**PHP version:** 7.3

**Server version:** 16

Is there a way to change this? Is it planned?

Bye Defcon0

**Comment by georgehrke** [comment]
Duplicate of https://github.com/nextcloud/calendar/issues/1329
